MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+include CMakeLists.txt
+recursive-include CMakeModules *
+include LICENSE
```

Comment:
Aren't you forgetting `README.md`?

Reply:
It's already included by default, but I am going to add it now anyway